src/lib/util.ts

```diff
@@ -159,3 +159,24 @@ export function inputFieldValueAsNumber(field: any): number {
 export function inputFieldCheckedAsNumber(field: any): number {
   return Number((field as HTMLInputElement).checked);
 }
+
+/**
+ * Creates a promise that resolves when the user types the konami code.
+ */
+export function konami(): Promise<void> {
+  return new Promise((resolve) => {
+    const expectedPattern = '38384040373937396665';
```

Comment:
`// ↑	↑	↓ ↓ ← → ← → B A`

Reply:
I was a little conflicted in terms of comments. As in, would it make it too obvious? But I guess it isn't supposed to be that secret. I'll add the comment.